kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/PropertiesController.java

```diff
@@ -105,25 +107,23 @@ private void setupShowingPanelHandlers() {
 
         editFieldsJfxNode = FXMLMvvmLoader.make(config);
 
-        Config closePropertiesConfig = new Config(this.getClass().getResource("close-properties.fxml"))
-                .addNamedViewModel(new NamedVm("genEditingViewModel", genEditingViewModel));
-        JFXNode<Pane, ClosePropertiesController> closePropsJfxNode = FXMLMvvmLoader.make(closePropertiesConfig);
+        ConfirmationPaneCommonViewModel confirmationViewModel = new ConfirmationPaneCommonViewModel();
+
+        Config closePropertiesConfig = new Config(this.getClass().getResource("/dev/ikm/komet/kview/mvvm/view/confirmation/confirmation-pane-common.fxml"))
+                .addNamedViewModel(new NamedVm("viewModel", confirmationViewModel));
+        JFXNode<Pane, ConfirmationPaneCommonController> closePropsJfxNode = FXMLMvvmLoader.make(closePropertiesConfig);
         closePropsPane = closePropsJfxNode.node();
-        closePropertiesController = closePropsJfxNode.controller();
+
+        confirmationViewModel.setNotificationTopic(genEditingViewModel.getPropertyValue(WINDOW_TOPIC));
+        confirmationViewModel.setNotificationEvent(new PropertyPanelEvent(closePropsPane, CLOSE_PANEL));
 
         genEditingEventSubscriber = evt -> {
             LOG.info("Publish event type: " + evt.getEventType());
 
             // "Semantic Details Added" is displayed when form values are Submitted when in CREATE mode
             // "Semantic Details Changed" is displayed when form values are Submitted when in EDIT mode
 
-            // TODO Changed is future, waiting for Figma wireframe
-
-//            if (genEditingViewModel.getPropertyValue(MODE).equals(CREATE)) {
-                closePropertiesController.showSemanticDetailsAdded();
-//            } else {
-//                closePropertiesController.showSemanticDetailsChanged();
-//            }
+            confirmationViewModel.setConfirmationMessage(ConfirmationMessages.SEMANTIC_DETAILS_ADDED);
```

Comment:
Can you call `confirmationViewModel.setPropertyValue(CONFIRMATION_MESSAGE, "the message goes here");`
No need to create a class with all the messages ( `ConfirmationMessages.SEMANTIC_DETAILS_ADDED`).

**Reason:** Since the confirmation controller is reusable across all these kinds of windows (aka Properties bump outs toggle button) message text could be whatever the developer decides. e.g. Lidr window, or Concept window they also have confirmation panes based on their transactions.  Since the genediting property controller owns the event subscriber (listener to adding or updating a semantic) it should own the message to be passed into the (reusable) confirmation pane controller's view model.